docs/source/tracking.rst

```diff
@@ -186,13 +187,29 @@ Example usage:
 
 .. _tracking_ui:
 
+Adding Tags to Runs
+~~~~~~~~~~~~~~~~~~~
+
+The :py:func:`mlflow.tracking.MlflowClient.set_tag` function lets you add custom tags to runs. For example:
+
+.. code:: python
+
+  client.set_tag(run.info.run_uuid, "tag_key", "tag_value")
+  
+If you log a run from a Databricks notebook, the run has the default tags::
+  
+  mlflow.databricks.notebookID
```

Comment:
Just one question, is there a reason we're calling these tags out explicitly? It might be a bit tough to keep this list in sync, maybe instead we could state that 


> "MLflow automatically logs some tags under the 'mlflow' namespace. For example, run names are logged as a tag with name 'mlflow.runName'. Thus we recommend against using tag names beginning with 'mlflow'"

Reply:
I mainly added them because folks see them in the MLflow UI.

Reply:
Ah got it, as of MLflow 0.7.0 internal tags are hidden in the UI, so maybe we can just leave them undocumented (or warn users not to log tags with the "mflow" prefix)

Reply:
Done in latest commit.